### DISCUSSION TITLE
can I turn off the TUnit logo when running tests?

### DISCUSSION THREAD

**Discussion by jamesshenry** [thomhurst/TUnit#692] [General]
The TUnit figlet text takes up too many lines for me to use `dotnet watch` effectively

**Comment by thomhurst** [comment]
> The TUnit figlet text takes up too many lines for me to use `dotnet watch` effectively

I can add a CLI flag to not print it. Will that suffice?

**Reply by jamesshenry** [reply]
yes please that would be ideal!

**Reply by thomhurst** [reply]
@henry-js this should be possible now in v0.1.894 by passing in the CLI argument `--disable-logo`
